src/common/_AllGoodsSlides/_AllGoodsSlides.tsx

```diff
@@ -0,0 +1,37 @@
+import React from "react"
+import { GoodCard, GoodCardFooter, GoodCardHeader } from "../GoodCard/GoodCard"
+
+export function _AllGoodsSlides({slidesData}) {
+  return (
+    <>
+      {slidesData.map(slide =>
```

Comment:
we use loop here and don't use `key` in GoodCard. And why component name starts with ''_"?